aws-athena/views/default-vw_pin_history.sql

```diff
@@ -2,15 +2,18 @@
 SELECT
     vwpv.pin,
     vwpv.year,
-    par.class,
+    REGEXP_REPLACE(par.class, '([^0-9EXR])', '') AS class,
```

Comment:
**question (blocking):** What was the motivation for this regex? Were there non `EX`, `RR`, or numeric classes in the data?

Reply:
yes, '2-99'